spec.bs

```diff
@@ -260,8 +260,21 @@ dictionary AuctionAdInterestGroupKey {
 The <dfn for=Navigator method>leaveAdInterestGroup(group)</dfn> method steps
 are:
 
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the
+  "[=join-ad-interest-group=]" [=policy-controlled feature=], then [=exception/throw=] a
+  "{{NotAllowedError}}" {{DOMException}}.
+1. Let |ownerUrl| be the result of running the [=URL parser=] on
+    |group|["{{AuctionAdInterestGroupKey/owner}}"].
+1. If |ownerUrl| is an error [=exception/throw=] a {{TypeError}}.

```

Comment:
The URL Parser returns "failure", so it's probably more correct to say to "is failure".
```suggestion
1. If |ownerUrl| is failure, [=exception/throw=] a {{TypeError}}.

```